staging/src/k8s.io/code-generator/cmd/conversion-gen/main.go

```diff
@@ -51,14 +51,6 @@ func main() {
 	// TODO: make callers pass this in.  It is too opaque here, and any use of
```

Comment:
A similar TODO should follow the code that inserts magic - can you put a comment in the new place and strike this?

Reply:
I have split the peer dirs into base-peer-dirs (apimachinery only) and extra-peer-dirs. The former are basically constant for everybody. The later are api specific, e.g. Kubernetes. For the very rare situations that you want to modify the base-peer-dirs as well, you can. But we can maintain that list without breaking people using extra-peer-dirs.